addons/binding/pom.xml

```diff
@@ -42,6 +42,7 @@
     <module>org.openhab.binding.fronius</module>
     <module>org.openhab.binding.ftpupload</module>
     <module>org.openhab.binding.gardena</module>
+    <module>org.openhab.binding.groheondus</module>
```

Comment:
Please also add a feature for the groheondus binding at the right alphabetical position in `features/openhab-addons/src/main/feature/feature.xml`. Without this feature the binding will not show in the available binding list and people can not use it.

Reply:
Done!